src/layout/css/style.scss

```diff
@@ -69,72 +94,87 @@ span.error, span.failed, span.xpassed, .error .col-result, .failed .col-result,
  *------------------*/
 
 #results-table {
-	border: 1px solid #e6e6e6;
+	border: $border-width solid #e6e6e6;
 	color: #999;
-	font-size: 12px;
-	width: 100%
-}
+	font-size: $font-size-text;
+	width: 100%;
 
-#results-table th, #results-table td {
-	padding: 5px;
-	border: 1px solid #E6E6E6;
-	text-align: left
-}
-#results-table th {
-	font-weight: bold
+	th,
+	td {
+		padding: $spacing;
+		border: $border-width solid #E6E6E6;
+		text-align: left;
+	}
+
+	th {
+		font-weight: bold;
+	}
 }
 
 /*------------------
  * 2. Extra
  *------------------*/
 
-.log:only-child {
-	height: inherit
-}
+$extra-spacing: $spacing;
+$extra-height: 240px;
+$extra-media-ratio: 4 / 3;
+$extra-media-width: round($extra-height * $extra-media-ratio);
```

Comment:
Same comment about expressing `$extra-media-width` in terms of a relation of `$extra-height` and `$extra-media-ratio`. As above

Reply:
done